From 170921d9e391789dc3a4041eb4685ffec6e58d30 Mon Sep 17 00:00:00 2001
From: Kyle McMartin <kyle@mcmartin.ca>
Date: Fri, 29 Oct 2010 12:48:01 -0400
Patch-mainline: 2.6.37-rc1
References: fate#310916
Git-commit: 6a915c2bf073743dd31229f1ec2eaa7a2b13c1c3
Subject: [PATCH 2373/2588] parisc-agp: fix missing slab.h include

Commit 338e4fab added a missing kfree if the alloc_pci_dev failed
but forgot to include <linux/slab.h> for the definition of
kfree.

Signed-off-by: Kyle McMartin <kyle@redhat.com>
(cherry picked from commit 6a915c2bf073743dd31229f1ec2eaa7a2b13c1c3)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/char/agp/parisc-agp.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/char/agp/parisc-agp.c b/drivers/char/agp/parisc-agp.c
index 17e380f..94821ab 100644
--- a/drivers/char/agp/parisc-agp.c
+++ b/drivers/char/agp/parisc-agp.c
@@ -19,6 +19,7 @@
 #include <linux/klist.h>
 #include <linux/agp_backend.h>
 #include <linux/log2.h>
+#include <linux/slab.h>
 
 #include <asm/parisc-device.h>
 #include <asm/ropes.h>
-- 
1.7.6

